frontend/src/scenes/funnels/funnelLogic.test.ts

```diff
@@ -0,0 +1,188 @@
+import { BuiltLogic } from 'kea'
+import { funnelLogic } from './funnelLogic'
+import { funnelLogicType } from './funnelLogicType'
+import { api, mockAPI } from 'lib/api.mock'
+import { expectLogic, initKeaTestLogic } from '~/test/kea-test-utils'
+import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
+import { preflightLogic } from 'scenes/PreflightCheck/logic'
+import { funnelsModel } from '~/models/funnelsModel'
+import { insightLogic } from 'scenes/insights/insightLogic'
+import { insightHistoryLogic } from 'scenes/insights/InsightHistoryPanel/insightHistoryLogic'
+
+jest.mock('lib/api')
+
+describe('funnelLogic', () => {
+    let logic: BuiltLogic<funnelLogicType>
+
+    mockAPI(async ({ pathname }) => {
+        if (pathname === 'api/insight/funnel/') {
+            return {
+                is_cached: true,
+                last_refresh: '2021-09-16T13:41:41.297295Z',
+                result: [],
+                type: 'Funnel',
+            }
+        } else if (pathname.startsWith('api/insight')) {
+            return { results: [], next: null }
+        } else if (pathname === '_preflight/') {
+            return { is_clickhouse_enabled: true }
+        } else if (pathname === 'api/users/@me/') {
+            return {}
+        } else {
+            console.error(`Unmocked fetch to: ${pathname}`)
+            debugger
```

Comment:
What does `debugger` do in here?

Reply:
Why it drops you into a debugger of course! Removed.